src/RZA1/diskio.c

```diff
@@ -203,7 +203,13 @@ DRESULT disk_read_without_streaming_first(BYTE pdrv, /* Physical drive nmuber to
     BYTE err;
 
     if (currentlyAccessingCard)
-        freezeWithError("E259"); // Operatricks got! But I think I fixed.
+    {
+        if (ALPHA_OR_BETA_VERSION)
+        {
+            // Operatricks got! But I think I fixed.
+            FREEZE_WITH_ERROR("E259");
```

Comment:
I think this should be unguarded? looks like it could lead to file corruption if ignored

Reply:
That check was previously here https://github.com/SynthstromAudible/DelugeFirmware/pull/690/files#diff-130d77e2bdfd7b590ceb981f0e747fabfa079b3ec5b547e8117108db13dee193L83 which is repurposed so the logic didn't change